models/eos/acl/eos_acl.yml

```diff
@@ -0,0 +1,224 @@
+---
+GENERATOR_VERSION: '1.0'
+
+ANSIBLE_METADATA: |
+    {
+      'metadata_version': '1.1',
+      'status': ['preview'],
+      'supported_by': 'network'
+    }
+NETWORK_OS: eos
+RESOURCE: acl
+COPYRIGHT: Copyright 2019 Red Hat
+
+DOCUMENTATION: |
+  module: eos_acl
+  version_added: 2.10
+  short_description: 'Manages ip access-list attributes of Arista EOS interfaces'
+  description: This module manages the ip access-list attributes of Arista EOS interfaces.
+  author: Gomathiselvi S (@GomathiselviS)
+  notes:
+  - Tested against Arista vEOS v4.20.10M
+  options:
+    config:
+      description: A dictionary of ip access-list options
+      type: list
+      elements: dict
```

Comment:
There should be a top-level key indicating the address family to which the access list belongs. Otherwise, we wouldn't be able to differentiate between an IPv4 and IPv6 ACL.